docs/features/archival.md

```diff
@@ -55,7 +51,7 @@ Then create a trust relationships for your EKS cluster:
             "Action": "sts:AssumeRoleWithWebIdentity",
             "Condition": {
                 "StringEquals": {
-                    "oidc.eks.<aws_region>.amazonaws.com/id/<cluster_id>:sub": ["system:serviceaccount:<temporal_ns>:<temporal_sa>"]
+                    "oidc.eks.<aws_region>.amazonaws.com/id/<cluster_id>:sub": ["system:serviceaccount:<temporal_ns>:<temporal_history_sa>"]
```

Comment:
Not very sure about this. Please double check.

Reply:
This is the same SA for all services, so no need to specify "history"

Reply:
Is it required by all service account?

Reply:
It seems each service has its own service account.

```
kubectl get sa -n temporal-tok
NAME                    SECRETS   AGE
default                 0         19h
temporal-tok-frontend   0         61m
temporal-tok-history    0         61m
temporal-tok-matching   0         61m
temporal-tok-worker     0         61m
```

Reply:
Sorry, answered to quickly! You're right, there is one SA per service! 
All services have the `eks.amazonaws.com/role-arn` annotation. I don't know which service is responsible to connect to s3 for archival. I'll try to find the info!

Reply:
@yujunz  @alexandrevilain  History service responsible for archive workflow

Reply:
![image](https://github.com/alexandrevilain/temporal-operator/assets/21026249/77471bbc-dc5a-4210-b2e2-cbb1f7277a53)

Reply:
at feature release, need to add worker SA if use aws IAM access to OpenSearch/RDS

Reply:
So let's merge this PR, thanks @DodgeCamaro !